### DISCUSSION TITLE
如何开启端口？例如 7890、7891 等常见代理设置。

### DISCUSSION THREAD

**Discussion by wclebb** [vernesong/OpenClash#2996] [Q&A] [answered]
搜了一遍似乎没看明白，我之前有个旧版本是可以直接使用 7890 7891 等常见代理设置的，即使使用 [switchyomega](https://chrome.google.com/webstore/detail/proxy-switchyomega/padekgcemlokbadohgkifijomclgjgif) 也能代理。

后来由于某些小bug，想尝试新版本是否解决这个问题。
于是尝试新盘安装（非升级），然后如果使用设定这个 IP 为网关，DNS 设置一样的 IP，则能代理。
比如 OpenWrt IP 为 192.168.1.2，别的设备网关设定192.168.1.2，DNS 192.168.1.2 即可代理。
似乎只能走网关 DNS 服务器才能代理，所以像是 7890、7891 等端口因为连不上，无法代理，我猜是因为端口没开放。

然后如果使用 switchyomega 设置相关的 7890 7891 等相关的端口，浏览器和 NAS 均无法代理。

搜了好多个不同的词，但仍然没看明白，好像是 OpenWrt 的防火墙，也好像是 OpenClash 自己的端口等等。
能否简单给个指明 URL 有教程或关键字，让 7890 7891 等相关端口得到开放并代理。

另说明：是用来做旁路有。

**Comment by walker2top** [comment, marked as answer]
碰到了同样的问题，最后是把OpenClash自动为 SOCKS5/HTTP(S) 代理设置的账户密码取消之后可以成功让浏览器走代理（用7893混合端口）。你可以试一下

**Comment by 369696738** [comment]


**Reply by walker2top** [reply]
覆写设置，拖到最下面有个“设置 SOCKS5/HTTP(S) 认证信息”

**Reply by peoplechinapower** [reply]
感谢 你的回答帮到了我

**Reply by K23Flux** [reply]
> 覆写设置，拖到最下面有个“设置 SOCKS5/HTTP(S) 认证信息”

感谢兄弟 帮了我大忙

**Reply by chenzz** [reply]
帮了大忙，谢谢！

**Reply by z2282023838** [reply]
谢谢

**Reply by AlfonsoLu** [reply]
🙏Thank you!

**Reply by Gtchen812** [reply]
感谢老铁 试了几天 解决问题了

**Reply by nerazzurri0302** [reply]
感谢，正在研究这个问题，解决了

**Comment by 1tonsilla** [comment]
太有帮助了，感谢，本来emby使用旁路由削刮的，但是一直无法搜索，不知道为啥。emby使用我电脑clash分享的端口是可以削刮的；然后旁路由整了半天发现都不行。最后搜到这个帖子，原来关闭认证就行了！

**Comment by tianma112** [comment]
感谢,困扰了几天的问题
